From d528a992f3f828261cae10d2a42fd575d0f9db03 Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Sun, 18 Mar 2018 00:01:05 +0100
Subject: [PATCH] makepkg: workaround man bugs

* looks like some man pages , most the one are
* created from xml , are weird formated..
* until we find out we use and workaround..
---
 scripts/makepkg | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/makepkg b/scripts/makepkg
index 97cb9408..6dddbfba 100755
--- a/scripts/makepkg
+++ b/scripts/makepkg
@@ -2088,6 +2088,11 @@ find $startdir/{pkg,pkg.*}/{usr{,/local,/share},opt/*}/{info,man} -type f 2>/dev
 			rm -f "$ln"
 			ln -sf "${fn}.gz" "${ln}.gz"
 		done
+		# we have some weird formatting bug for xml
+		# and some custom made ones. untill we figure
+		# workaround it - crazy -
+		# drop spaces , tabs and all empty lines..
+		sed -i -e 's/^[ \t]*//' -e '/^$/d' "$i"
 		# compress the original
 		gzip -9 "$i"
 	fi
-- 
2.16.2

